packages/proxy/lib/http/index.ts

```diff
@@ -182,7 +183,13 @@ export function _runStage (type: HttpStages, ctx: any, onError: Function) {
       const fullCtx = {
         next: () => {
           fullCtx.next = () => {
-            throw new Error('Error running proxy middleware: Cannot call this.next() more than once in the same middleware function. Doing so can cause unintended issues.')
+            const error = new Error('Error running proxy middleware: Detected `this.next()` was called more than once in the same middleware function. This can cause unintended issues and may indicate an error in your middleware logic or configuration.')
```

Comment:
I think this should be less verbose - at least, we shouldn't mention "configuration" here - this is an internal error that represents an error in our codebase, and seeing "configuration" could lead a user to think that they need to change their `cypress.config.js`.

```suggestion
            const error = new Error('Error running proxy middleware: Detected `this.next()` was called more than once in the same middleware function, but a middleware can only be completed once.')
```